app/components/form-input.hbs

```diff
@@ -9,6 +10,13 @@
     @class={{if @showError 'input errorBorder' 'input'}}
     {{on 'input' (fn this.inputFieldChanged)}}
   />
+  {{#if @buttonupload }}
+  <button class="upload" type="submit">update location</button>
```

Comment:
NIT: update location -> Update, as the user already knows they are updating the location from the Input Label